pupil-hardware/pupil-headset.md

```diff
@@ -17,89 +17,65 @@ Pupil ships with a number of additional parts. The below sections provide an ove
-
-> <p style="text-align: center;">Adjust the ball joint firmness by adjusting the socket screw.</p>
-
-<div class="content-container" style="clear:none;">
-  <div class="Grid Grid--1of3 u-textCenter">
-    <div class="Grid-cell">
-      {{< figure-img figure-class="figure-container" src="/images/pupil-hardware/eyearm.gif" >}}
-      <p style="padding: 10px;">You can <b>slide</b> the eye camera arm along the track.</p>
-    </div>
-    <div class="Grid-cell">
-     {{< figure-img figure-class="figure-container" src="/images/pupil-hardware/eyerotate.gif" >}}
-      <p style="padding: 10px;">You can <b>rotate</b> the eye camera about its ball joint.</p>
-    </div>
-    <div class="Grid-cell">
-      {{< figure-img figure-class="figure-container" src="/images/pupil-hardware/world.gif" >}}
-      <p style="padding: 10px;">You can <b>rotate</b> the world camera to align with your FOV.</p>
-    </div>
-  </div>
-</div>
+A lot of design and engineering thought has gone into getting the ergonomics of the headset just right. It is designed to fit snugly, securely, and comfortably. The headset and cameras can be adjusted to accommodate a wide range of users. 
+
+To ensure a robust eye tracking performance, make sure all the cameras are in focus with a good field of view of your eyes.
+
+#### Slide
+> {{< video-webm src="/videos/headset-adjust/eyecam-slide.webm" >}}
+
+The eye camera arm slides in and out of the headset frame. You can slide the eye camera arm along the track.
+
+#### Rotate
+> {{< video-webm src="/videos/headset-adjust/worldcam-rotate.webm" >}}
+
+You can rotate the world camera up and down to align with your FOV.
+
+#### Rotate
+> {{< video-webm src="/videos/headset-adjust/eyecam-rotate.webm" >}}
+
+The eye camera arm is connected to the eye camera via the ball joint. You can rotate about its ball joint.
+
+#### Screw
+> {{< video-webm src="/videos/headset-adjust/eyecam-screw.webm" >}}
+
+The ball joint controls the firmness of the eye camera. Adjust the socket screw to increase or decrease the firmness.
 
 <aside class="notice">
   Before you calibrate, make sure to properly adjust the headset to get optimal eye detection and world camera FOV.
 </aside>
 
 ### Focus Cameras
-
 #### Focus Eye Camera
-> {{< lqip-img figure-class="img-m" src="/images/pupil-hardware/pupil-parts-2.jpg" >}}
-
-**Very important** - make sure the eye camera is in focus. If you can see details of your iris, then the focus is most likely good. **Slide the eye camera arm** to adjust focus and/or use the lens adjuster tool.
-
-#### Focus World Camera
-> {{< lqip-img figure-class="img-m" src="/images/pupil-hardware/pupil-parts-4.jpg" >}}
-
-Set the focus for the distance at which you will be calibrating.
 
+> {{< video-webm src="/videos/headset-adjust/eye-adjust.webm" >}}
 
+Make sure the eye camera is in focus. If you can see details of your iris, then the focus is most likely good. **Slide the eye camera arm** to adjust focus and/or use the lens adjuster tool.
```

Comment:
Second sentence revision. 

Twist the lens focus ring of the eye camera with your fingers or lens adjuster tool to bring the eye camera into focus.